sklearn/model_selection/_split.py

```diff
@@ -257,7 +257,7 @@ def get_n_splits(self, X, y=None, labels=None):
 
 
 class _BaseKFold(with_metaclass(ABCMeta, BaseCrossValidator)):
-    """Base class for KFold and StratifiedKFold"""
+    """Base class for KFold, LabelKFold and StratifiedKFold"""
```

Comment:
should have a comma after LabelKFold (`KFold, LabelKFold and StratifiedKFold` -> ``KFold, LabelKFold, and StratifiedKFold`)